library/psa_crypto.c

```diff
@@ -210,13 +209,16 @@ psa_status_t mbedtls_to_psa_error( int ret )
         case MBEDTLS_ERR_CMAC_HW_ACCEL_FAILED:
             return( PSA_ERROR_HARDWARE_FAILURE );
 
+#if !( defined(MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG) ||      \
+       defined(MBEDTLS_PSA_HMAC_DRBG_MD_TYPE) )
```

Comment:
I think is clearer to use `MBEDTLS_CTR_DRBG_C` here. These error code should only be available when the CTR_DRBG module is enabled.
```suggestion
#if !defined(MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG) &&      \
       defined(MBEDTLS_CTR_DRBG_C) )
```

Reply:
`MBEDTLS_CTR_DRBG_C` is actually not the correct condition since it may be included in the build but not used from PSA crypto. I've added a comment to explain the condition. Likewise for HMAC_DRBG below.